doomed_patient/doomed_patient/PatientGame.cpp

```diff
@@ -107,7 +115,9 @@ void PatientGame::handleEvents()
 // Game updates that need to happen every frame go here
 void PatientGame::update()
 {
-
+	// Temporary before collision checking is implemented
+	if (player->getCentre() == exit->getCentre())
```

Comment:
This is just temporary until we implement collision checking (at the moment the player's centre needs to be in the same position as the exit's centre!)